src/main/java/com/openshift/jenkins/plugins/pipeline/OcAction.java

```diff
@@ -195,11 +197,9 @@ protected OcActionResult run() throws Exception {
                 ByteArrayOutputStream stdErr = new ByteArrayOutputStream();
                 ByteArrayOutputStream stdOut = new ByteArrayOutputStream();
 
-                exitStatusRaceConditionBugWorkaround( dtc, filePath, launcher);
-
                 long reCheckSleep = 250;
                 Integer exitStatus;
-                while ( ( exitStatus = dtc.exitStatus(filePath,launcher) ) == null ) {
+                while ( ( exitStatus = exitStatusRaceConditionBugWorkaround(dtc, filePath,launcher) ) == null ) {
```

Comment:
To clarify, per my comment at https://github.com/openshift/jenkins-client-plugin/pull/29#issuecomment-305353125 .... `dtc.exitStatus(filePath,launcher)` returns null while the task is in progress and until that task finishes one way or the other